Add CSS "background-attachment: fixed" for initial parallax feature

NickFoden opened the pull request.

NickFoden committed:
Add CSS background-attachment fixed for initial parallax feature

donavon commented:
:heart: it!

donavon commented:
There seem to be a few of changes to the CSS file that appear to be unrelated to your solution. It's making it hard to focus on just the applicable changes. I'm also concerned that the other changes may introduce new unintended issues. 

However, I ran the code and it seems fine.

NickFoden commented:
I added one line of css to the .hero class (background-attachment: fixed ) and then alphabetized values in the css .hero class because of my preference for ordering by abc. Will try to avoid that in future to enable clear review. 

All other changes are my prettier i think remove leading 0's and simplifying padding and margin values to be the shorthand. In future I'll disable vs code format on save and only change more specific concise pieces.
